LoopKit/CorrectionRangeOverrides.swift

```diff
@@ -44,3 +50,70 @@ public extension CorrectionRangeOverrides.Preset {
         }
     }
 }
+
+extension CorrectionRangeOverrides: Codable {
```

Comment:
Perhaps we should consider some custom types `GlucoseValue` and `GlucoseValueRange` to make it explicitly clear that it isn't just some random range of *any* `HKQuantity`, but one exactly used for glucose values and only has valid units of mg/dL and mmol/L.

Then we could shove the coding into that type along with the units.

Reply:
And I think we could then get rid of this `CorrectionRangeOverride` struct.

Reply:
But perhaps this is a large topic of discussion.